changelog/1123.trivial.rst

```diff
@@ -0,0 +1,2 @@
+Changed |ParticleList| so that if it is provided with no arguments, then it creates
+an empty |ParticleList|.  This behavior is analogous to how `list` and `tuple` work.
```

Comment:
Before, this probably raised an exception, right? Asking because that'd make this a breaking change.

Reply:
I had been wondering about whether or not something like this would be considered a breaking change, but now that you mention it, I agree with making it a breaking change.